run.py

```diff
@@ -26,6 +56,15 @@ def run():
     # Open the file once it has been downloaded
     subprocess.call(["open", filename])
 
+    # Open the file once it has been downloaded (on Mac os)
+    try:
+        subprocess.call(["open", filename])
+    except FileNotFoundError:
+        # this will take care of the windows part
+        subprocess.call(('cmd', '/C', 'start', '', filename))
+
```

Comment:
I think it's better if you explicitly check whether the user is using Windows, you can do this with the OS library: https://stackoverflow.com/questions/1325581/how-do-i-check-if-im-running-on-windows-in-python

Reply:
That was helpful.  It was cool to use `os.path()` on windows and `subprocess.call()` on `mac` os.